### ISSUE TITLE
Time-Parsing with `provideLuxonDatetimeAdapter`

### ISSUE THREAD

**Issue by angelaki** [ng-matero/extensions#308]
In your demo (https://ng-matero.github.io/extensions/components/datetimepicker/overview) I can put quite simple formats and the component / the adapter tries it's best to find the correct value (e.g. time-input _7_ gets pared to _07:00_. But using your `LuxonDatetimeAdapter` it seams like I need to put in the absolute correct format.

Is this an issue of this adapter? Or am I configuring something wrong?

**Comment by nzbin** [comment]
Can you provide an online example?

**Comment by angelaki** [comment]
Just cloned your repo and just needed to change that single line:

https://github.com/ng-matero/extensions/blob/18cf6d5a53240af533bd7447e28666bf1c6b3bcc/projects/dev-app/src/app/datetimepicker/datetimepicker-demo.component.ts#L54

Replace it with `provideLuxonDatetimeAdapter` and you'll notice that from now on you need to pass the exact format on input while with the `provideMomentDatetimeAdapter` the input gets parsed / auto-corrected.

**Comment by angelaki** [comment]
@nzbin yeah, sorry. Should have noted that format doesn't make a difference. If you just put `provideMomentDatetimeAdapter()` (falling back to default settings) vs. `provideLuxonDatetimeAdapter()` (same) Moment works but Luxon doesn't.
